exercises/ch04/exercise_a.js

```diff
@@ -1,4 +1,9 @@
 // Refactor to remove all arguments by partially applying the function.
 
 // words :: String -> [String]
-const words = str => split(' ', str);
+// Original code
+// const words = str => split(' ', str);
+//
+
+// const words = str => split(' ', str);
+const words = split(' ');
```

Comment:
Pointfree style is only mentioned in Chapter 5, so the code is purposely written in a non-pointfree style here. The change is therefore ill-advised.